### PR TITLE
Set up a "Hello Microverse" project

### PR DESCRIPTION
**Pull request by edubew**
### Linters installation
**In this milestone, I was able to:**

- Created hello microverse project
- Installed Linters
- Created HTML file, CSS file, and test.md file and ignored the test.md file
- Modified the README file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse!</title>
```
Comment (Deepakdanger) [inline index.html:7]: - [ ] **Required** Let's add the `CSS` file link here so that the CSS code will reflect here.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @edubew, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Good PR ✔️ 
- Good commit ✔️ 
- No linter error ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,47 @@
...
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/edubew)
+- Twitter: [@twitterhandle](https://twitter.com/@edube_winne)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/ winfred edube)
```
[the remark this answers is not in the document]
Comment (edubew) [reply README.md:27]: Hello ShoiraTa, How is the link supposed to be? I checked it before requesting a review and it still isn't correct.

[the remark this answers is not in the document]
Comment (ShoiraTa) [reply README.md:27]: Hi, Please check your linkedin, you should have the link to your profile there. go to your profile then copy the url, this is my example:
Screenshot 2022-02-10 at 22 42 46
 As you can see, my url is `https://www.linkedin.com/in/shoira-ta/`

[the remark this answers is not in the document]
Comment (Whoistolu) [reply README.md]: 👍🏽 All good now.

**Review by Whoistolu — APPROVED** [review]
### Hi @edubew,

Your project is complete!  There is nothing else to say other than... It's time to merge.

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**